in_toto/util_darwin.go

```diff
@@ -0,0 +1,11 @@
+package in_toto
```

Comment:
Curious, but could we combine this and `_linux` into one file, and use a `!windows` constraint instead of maintaining two copies of the same function?

Reply:
Apart from that, I think this is all fine. :smile:

Reply:
The `!windows` constraint is file based.. we would have two different files again. One for Linux and one for Windows. 

Or am I just not understanding what you mean?

Reply:
Yes, but currently we have three, `_linux`, `_darwin`, and `_windows`. The first two are identical as well. We can use a single file there with the `!windows` constraint?

Reply:
ah, this is what you mean. Yes, this is possible! Maybe even _unix works as suffix.

Reply:
@adityasaky  check the latest commit.. fixed